docs/pages/zero-trust-access/compliance-frameworks/fedramp.mdx

```diff
@@ -6,6 +6,7 @@ tags:
  - conceptual
  - zero-trust
  - audit
+enterprise: FedRAMP compliance
```

Comment:
This item is correct, but it looks like we missed the SOC2 document below this in the menu.